src/devices/HardwareMonitor/SensorJson.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
consider moving all `*Json.cs` files to a separate folder like: `model/` or `JsonModel/`, moving them to a separate namespace and removing prefixes